geminimd/render.py

```
@@ -0,0 +1,319 @@
+        md_parts.append(f"sourceCreatedTime: {created_time}\n")
+    if source_mime:
+        md_parts.append(f"sourceMimeType: {source_mime}\n")
+    if source_size is not None:
+        md_parts.append(f"sourceSizeBytes: {source_size}\n")
+    if run_settings:
+        model = run_settings.get("model")
+        if model:
+            md_parts.append(f"model: \"{model}\"\n")
+        for k in ("temperature", "topP", "topK", "maxOutputTokens"):
+            if k in run_settings:
+                md_parts.append(f"{k}: {run_settings[k]}\n")
```

Ruff (0.15.9)

[warning] Use `list.extend` to create a transformed list

(PERF401)